src/adapters/stubAdapter.js

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+
+const BaseAdapter = require('../adapter');
+
+class StubAdapter extends BaseAdapter {
```

Comment:
Yeah, just something that doesn't make any external calls so we can test apply and toJSON in isolation